src/routes/routes.tsx

```
@@ -145,7 +152,11 @@
   },
   {
     path: 'org-admin',
-    element: <AuthProvider><PageWrapper children={<DashboardLayout />} /></AuthProvider>,
+    element: (
+      <AuthProvider>
+        <PageWrapper children={<DashboardLayout />} />
```

Biome

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)